libs/cart/src/drivers/magento/queries/set-guest-email.ts

```diff
@@ -0,0 +1,17 @@
+import gql from 'graphql-tag';
+
+import { cartFragment } from './fragments';
+
+export const setGuestEmail = gql`
+  mutation SetGuestEmail($cartId: String!, $email: String!) {
+    setGuestEmailOnCart(
+      cart_id: $cartId,
+      email: $email
+    ) {
+      cart {
+        ...cart
```

Comment:
As far as I can tell, we won't be using the `setGuestEmail` query alone, because daffodil will set the email when the address is set. Maybe this should just query for the email and not the whole cart? Because in the billing address driver, we're querying for the entire cart twice.

Reply:
Good point. Will change.

Reply:
Changed in [`0c43f4f`](https://github.com/graycoreio/daffodil/pull/671/commits/0c43f4f175486f52edefc55c0806641d76db840a).